syntax = "proto3";

service Shard {
  rpc Receive (ShardOuts) returns (Empty) {}
  rpc StartTest (Empty) returns (Empty) {}
}

message ShardOuts {
  string url = 1;
  int32 block_num = 2;
  bytes data = 3;
  bytes arr_map = 4;
}

message Empty {}
